### DISCUSSION TITLE
Tau-U Meta, Method "Complete" and Confidence Intervals Above 1

### DISCUSSION THREAD

**Discussion by gjones1219** [jazznbass/scan#90] [Q&A] [answered]
Hello again Jurgen! I hope you are taking good care. I had a quick question about the output for Tau-U meta.

In the "Jazz" book, it says that for Tau-U meta, method = "complete", values for for Tau-U output will range from [-1,1]. However, when one looks at the confidence intervals that are returned by the Tau-U meta function when using method = "complete", the values for the 95% Confidence Intervals are sometimes outside of this range (i.e., above and below [-1,1]). Please see the example in the scan book at this link: https://jazznbass.github.io/scan-Book/ch_overlapping_indices.html#tau-u (specifically, look at the output for the following line of code: tau_u(exampleAB) )

Can you please explain this discrepancy? Thanks so much!

Grant

**Comment by jazznbass** [comment]
Thank you!
There is a mistake here. I forgot to do the Fisher Z inversion transformation of the ci values.
I plan to release a new scan version in February when I have some time for the scan project.

**Reply by gjones1219** [reply]
Thank you. I would strongly recommend you also add a note ASAP to the Scan Book, as people will be getting incorrect results from the analyses they do with Tau-U meta and method = "Complete" at this point (as I almost did). I will then plan to switch the method to "Parker" -- can you confirm that this path forward will work for now and method = "parker" will return the correct results? Thank you.

**Reply by gjones1219** [reply]
In addition to the above, I noticed that method = "parker" seems to return Tau-U values that are actually bounded between [-1,1] but the confidence intervals do not seem to have the same bounds. Again, can you confirm the Tau-U function in this package is working properly or is there a way to generate the proper values here if not? Thanks.

**Reply by jazznbass** [reply]
I had implemented a method for calculating the confidence intervals that could lead to estimations for the upper boundary above 1.
Based on Long & Cliff (1997) I have now rewritten the procedure to give results within the boundary [-1;1]. These changes will be available in the next release.
For now, I would not recommend to report the confidence intervals as they are too conservative.

The switch to method = "parker" will not solve the original problem with the confidence intervals.
Actually, I am not sure if it possible to calculate confidence intervals for method = "parker" without implementing a time-consuming bootstrap method. 

Long, J. D., & Cliff, N. (1997). Confidence intervals for Kendall’s tau. British Journal of Mathematical and Statistical Psychology, 50(1), 31–41. https://doi.org/10.1111/j.2044-8317.1997.tb01100.x

**Reply by gjones1219** [reply]
As most journals require confidence intervals for results, I will plan to report them but note that the package uses a conservative method for estimating confidence intervals. Is there a citation or method that I can name for how the Scan package currently calculates confidence intervals so that any reviewers can be clear on the methodology?

I will look forward to the next release of Scan in February, and again just want to suggest adding a note to the book so that people are aware of all of this and aren't reporting incorrect results for method = "complete"

Thank you!

**Comment by jazznbass** [comment, marked as answer]
I turned this discussion into an issue
